app/screens/ScheduleScreen/ScheduleDayPicker.tsx

```diff
@@ -51,33 +49,34 @@ AnimatedDayButton.displayName = "AnimatedDayButton"
 type ScheduleDayPickerProps = {
   scrollX: SharedValue<number>
   onItemPress: (itemIndex) => void
-  schedules: Schedule[]
-  selectedSchedule: Schedule
+  scheduleDates: Date[]
+  selectedScheduleDate: Date
 }
 
 export const ScheduleDayPicker: FC<ScheduleDayPickerProps> = ({
   scrollX,
   onItemPress,
-  schedules,
-  selectedSchedule,
+  scheduleDates,
+  selectedScheduleDate,
 }) => {
   const wrapperWidth = width - spacing.extraSmall * 2
-  const widthSize = wrapperWidth / schedules.length
-  const index = schedules.findIndex((s) => s.date === selectedSchedule.date)
-
+  const widthSize = wrapperWidth / scheduleDates.length
+  const selectedScheduleIndex = scheduleDates.findIndex((date) =>
+    isSameDay(date, selectedScheduleDate),
+  )
```

Comment:
Renamed this so that it wasn't confused with all of the other uses of `index` within this file.